index.html

```diff
@@ -4,9 +4,58 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Google</title>
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
+    <nav>
+        <div id="left">
+            <a class="left" href="#">About</a>
+            <a class="left" href="#">Store</a>
+
+        </div>
+        <div id="right">
+            <a class="gmail" href="#">Gmail</a>
+            <a href="#">Images</a>
+            <img id="google-app" src="./googleapp.png" alt="Google apps " title="Google apps">
+            <button type="submit" class="btn">Sign In</button>
+
+        </div>
+    </nav>
+     
+    <div class="google_img">
+        <img src="https://web.archive.org/web/20191130234759im_/https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png"
+         alt="Google">
+
+    </div>
+    <div id="search" >
+        <span>
+            <img class="icon-search" src="./search_black_24dp.svg" alt="">
+        </span>
+        <input type="search"  >
+        <span>
+            <img src="./mic_black_24dp.svg" alt="">
+        </span>
+
+    </div>
+    <div class="two-btn">
+        <button type="submit" class="search-btn">Google Search</button>
+        <button type="submit" class="lucky-btn" >I'm Feeling Lucky</button>
+    </div>
+    <footer>
+        <div class="footer-left">
+            <a href="">Advertising</a>
+            <a href=""> Business</a>
+            <a href="">How search Works</a>
```

Comment:
we can used the <ul> tag and the <li> tag. Under that place the <a> tag to make it more semantic. Thank you.